### DISCUSSION TITLE
Self reset password is possible?

### DISCUSSION THREAD

**Discussion by zookato** [neos-sdi/adfsmfa#226] [Ideas]
Hey guys

Is it possible to add an ADFS function to Allow Self-Service Password Reset? It will be awesome if this feature is available.

**Comment by redhook62** [comment]
Yes it's possible.

On the other hand, we do not want to replace the dedicated administration tools.

This feature will be optional and disabled by default.

In addition, it will only be a question of resetting the account (must change password at next logon) which will also be subject to the same constraints as the administration tools (Member of Domain Administrators or member of Account Operators)

**Reply by redhook62** [reply]
Done in version 3.1.2205.0
